frontend/package.json

```diff
@@ -38,6 +38,7 @@
     "next": "12.3.0",
     "next-transpile-modules": "^10.0.1",
     "nextjs-google-analytics": "^2.3.3",
+    "package.json": "^2.0.1",
```

Comment:
we're not supposed to install any other package. if you could remove this that would be great